examples/vae/main.py

```diff
@@ -0,0 +1,147 @@
+from absl import app
+from absl import flags
+
+import jax.numpy as jnp
+import numpy as np
+
+import jax
+from jax import nn as jnn
```

Comment:
relu and sigmoid are available in nn already.

Reply:
Thank you for the comment.
I removed this line and use instead `nn.relu` and `nn.sigmoid`.